scripts/languages.ts

```diff
@@ -27,6 +27,14 @@ const EXTRA_SH_LANGUAGES: SupportLanguage[] = [
     filenames: ['.env.*'],
     vscodeLanguageIds: ['dotenv'],
   },
+  {
+    name: 'nvmrc',
+    since: '0.14.0',
+    parsers: ['sh'],
+    extensions: ['.nvmrc'],
+    filenames: ['.nvmrc'],
```

Comment:
Is `.node-version` the same thing?

Reply:
I think so! Will add.

Reply:
Or rather: it's similar and most tools I'm seeing now that use it treat it roughly the same.